src/Resources/config/doctrine/model/Brand.orm.xml

```diff
@@ -1,20 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<!--suppress XmlDefaultAttributeValue -->
 <doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                   xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
                                       http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
-    <mapped-superclass name="Loevgaard\SyliusBrandPlugin\Entity\Brand" table="loevgaard_brand">
+    <mapped-superclass name="Loevgaard\SyliusBrandPlugin\Model\Brand" table="loevgaard_brand">
         <id name="id" column="id" type="integer">
             <generator strategy="AUTO" />
         </id>
-        <one-to-many field="images" target-entity="Loevgaard\SyliusBrandPlugin\Entity\BrandImage" mapped-by="owner" orphan-removal="true">
+        <one-to-many field="images" target-entity="Loevgaard\SyliusBrandPlugin\Model\BrandImage" mapped-by="owner" orphan-removal="true">
             <cascade>
                 <cascade-all/>
             </cascade>
         </one-to-many>
-        <one-to-many field="products" target-entity="Sylius\Component\Product\Model\ProductInterface" mapped-by="brand" />
+        <one-to-many field="products" target-entity="Sylius\Component\Product\Model\ProductInterface" mapped-by="brand" fetch="EXTRA_LAZY" />
         <field name="slug" type="string" length="191" unique="true" nullable="false" />
```

Comment:
Should we add a code field or should we just keep the slug as the identifier?

Reply:
Somehow it probably would be more 'best practice' to add a code field

Reply:
Do you think we should remove the length attribute? Have you looked into the `utf8mb4` problem with lengths on VARCHARS?

Reply:
I believe we should not only add `code` rather than slug as identifier (Sylius best practice), but also make `name`, `slug` translatable. And probably add some (translatable, nullable) `description` which can be useful to display at front-end short information about brand on mouse over (or for other purposes, we shouldn't implement that front-end thing).

> utf8mb4
Lets keep it as is for now. I had low-priority task on other project to add support for emojis to existing database without success. If this required - I will dive deeper into utf8mb4.

Reply:
Here are my thoughts:

1. Remove length attribute since it seems that Sylius doesn't use this anywhere. If we ever run into some emoji problem we will figure it out then.
2. Add code field just to keep to Sylius best practices.
3. Do **not** make slug and name translatable. In this particular case we are talking about brands. Brands are the same in any language or at least they should be.
4. Do **not** add description field for now since it is a non breaking addition we can add it later if we need it

Reply:
Ok